postgrest_py/_async/client.py

```diff
@@ -20,16 +20,20 @@ def __init__(
         *,
         schema: str = "public",
         headers: Dict[str, str] = DEFAULT_POSTGREST_CLIENT_HEADERS,
+        timeout: Timeout = Timeout(5),
```

Comment:
I think we can just use an int instead of a Timeout object. That would easy our API usage. As far as I can remember, the Timeout object is used when you want to specify different Timeouts for read, write, connection, etc.

It's true that letting users use Timeout makes it more customizable, so maybe we could provide both options and make timeout a Union[Timeout, int].

Reply:
I also added the `float` type to the `Union`